### ISSUE TITLE
[Bug] defineAsCustomElements component

### ISSUE THREAD

**Issue by jrson83** [nanojsx/nano#69]
**Describe the bug**
No matter what I try I get this error when using customElements:

```
customElements.define(componentName, class extends HTMLElement {
    ^

ReferenceError: customElements is not defined
```
Here is a [Sandbox Link](https://codesandbox.io/s/runtime-hill-zcrx3?file=/src/pages/HomePage.tsx)

Also the usage of CustomElementsMode on the docs is missing some really usefull required information like import module, export component and actuall usage example.

**Have a question?**
Can you please fix it, or describe how to use it right if it's not a bug?
If the usage is working I would like to provide some more infos for the docs.

## Todo

- [x] #70
- [x] #71

**Comment by yandeu** [comment]
Thanks for reporting this issue.

`customElements` does only work in the browser  (https://developer.mozilla.org/en-US/docs/Web/API/Window/customElements)

You are also rendering `<HomePage />` on the server.

I guess I should make this component isomorphic.

- [ ]  Make defineAsCustomElements isomorphic.

**Comment by jrson83** [comment]
Ah right, I didn't realize it wasn't isomorphic. I setup with client only and it works.
I gonna experiment and then update the docs.

**Comment by jrson83** [comment]
Closed with 5656e30
